tests/test_onnx_export.py

```diff
@@ -663,7 +668,7 @@ def test_export_layernorm_linear(
             validate_result(fname, inp, model, atol=1e-3, is_fp8=use_fp8)
 
 
-@pytest.mark.parametrize("scale_factor", [112])
+@pytest.mark.parametrize("scale_factor", [[224, 224, 448, 448]])
```

Comment:
```suggestion
@pytest.mark.parametrize("scale_factors", [[224, 224, 448, 448]])
```

It will be helpful to add a line documenting which layers these scales affect.